adam-core/src/test/scala/org/bdgenomics/adam/models/CoverageSuite.scala

```diff
@@ -0,0 +1,112 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.models
+
+import org.bdgenomics.adam.util.ADAMFunSuite
+import org.bdgenomics.formats.avro.Feature
+
+class CoverageSuite extends ADAMFunSuite {
+  sparkTest("Convert to coverage from valid Feature") {
+    // create a valid feature
+    val featureToConvert =
+      Feature.newBuilder()
+        .setContigName("chr1")
+        .setStart(1)
+        .setEnd(2)
+        .setScore(100)
+        .build()
+
+    val coverageAfterConversion = Coverage(featureToConvert)
+
+    // all fields should match between the two
+    assert(coverageAfterConversion.start == featureToConvert.start)
+    assert(coverageAfterConversion.end == featureToConvert.end)
+    assert(coverageAfterConversion.contigName == featureToConvert.contigName)
+    assert(coverageAfterConversion.count == featureToConvert.score)
+  }
+
+  sparkTest("Convert to coverage from Feature with null/empty contigName fails with correct error") {
+    // feature with empty contigname is not valid when converting to Coverage
+    val featureWithEmptyContigName =
+      Feature.newBuilder()
+        .setContigName("")
+        .setStart(1)
+        .setEnd(2)
+        .setScore(100)
+        .build()
+
+    val caughtWithEmptyContigName =
+      intercept[java.lang.IllegalArgumentException](Coverage(featureWithEmptyContigName))
```

Comment:
Small nit, but you shouldn't need `java.lang.` in front of `IllegalArgumentException`. Would you mind making the change throughout the file?